### PR TITLE
Fixes for ppc64le (console and PXE) on PowerVM

### PR DESCRIPTION
**Pull request by sydefree**
1. Add hvc0 so that we can get the detailed installation messages for VMs' console on HMC.
2. Since ppc64le does not support regular PXE, we have to use grub2 network boot along with `core.elf` file.

### CODE REVIEW

**Comment by christianh814** [comment]
@sydefree In the future, can you please [PR against the `devel` branch](https://github.com/RedHatOfficial/ocp4-helpernode/blob/master/docs/contribute.md)? Thanks! I've already done it for your for this PR 

@bpradipt can you verify this change?

**Comment by bpradipt** [comment]
Sure @christianh814 I'll verify

**Comment by sydefree** [comment]
Sure @christianh814 !
My mistake. @bpradipt PXE seems more consitent with the classical way on ppc64le.

File: tasks/main.yml [hunk 1]
```diff
@@ -189,6 +189,12 @@
     shell: grub2-mknetdir --net-directory=/var/lib/tftpboot
     when: not staticips and ppc64le
 
+  - name: fake pxelinux.0 for ppc64le
+    copy:
+      src: /var/lib/tftpboot/boot/grub2/powerpc-ieee1275/core.elf
+      dest: /var/lib/tftpboot/pxelinux.0
+    when: not staticips and ppc64le
+
```
Comment (bpradipt) [inline tasks/main.yml:197]: @sydefree this will need a rebase due to https://github.com/RedHatOfficial/ocp4-helpernode/pull/62.

Comment (schabrolles) [reply tasks/main.yml:197]: Can I just remove those line and push again ?

Comment (schabrolles) [reply tasks/main.yml:197]: Can I just remove those lines and push agin ?

Comment (bpradipt) [reply tasks/main.yml:197]: Yeah..Please fetch the latest code and then remove the above lines. Just add the the console changes

**Comment by bpradipt** [comment]
I tested your changes - `console=tty0 console=ttyS0 console=hvc0,115200n8 ` successfully on different Power setups (KVM and PowerVM).  So once you resend the patch we should be good to go.

**Comment by schabrolles** [comment]
@bpradipt done ... `fake pxelinux for power`lines removed as it is already resolved by #62

**Pull request merged**